internal/configs/config_params.go

```diff
@@ -109,6 +110,7 @@ func NewDefaultConfigParams() *ConfigParams {
 		SSLPorts:                   []int{443},
 		MaxFails:                   1,
 		MaxConns:                   0,
+		UpstreamZoneSize: 			"256k",
```

Comment:
Looks like this is missing a run of `gofmt`, not gonna block the pr for it, but would be nice.

Reply:
`go fmt` should be run for consistency before merging